apps/asctb-reporter/src/app/components/compare/compare.component.html

```diff
@@ -32,12 +34,17 @@
                     #dataSource="matButtonToggleGroup"
                     (change)="onDataSourceChange(i)"
                   >
-                    <mat-button-toggle value="file" checked>Upload file</mat-button-toggle>
-                    <mat-button-toggle value="link">Link URL</mat-button-toggle>
+                    <mat-button-toggle value="file" checked hraFeature="upload-file" hraClickEvent
+                      >Upload file</mat-button-toggle
+                    >
+                    >
```

Comment:
I think there is an excess `>` here. Format the previous line better by putting a new lines between `>` and `Upload file` and `<`